modweaver.api/API/NewWeaponsManager.cs

```diff
@@ -0,0 +1,192 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using Unity.Netcode;
+using UnityEngine.UI;
+using UnityEngine;
+
+namespace modweaver.api.API {
+    public static class NewWeaponsManager {
+        private static uint _prefabHandlerId = 7777; //TODO: fix possible collision
```

Comment:
I'm not sure how we'll fix this- but I think it's fine for now. If we run into any issues then this'll need to be changed.